bioptim/limits/penalty.py

```diff
@@ -1244,6 +1244,11 @@ def _get_node(nlp, penalty: PenaltyOption):
                     x.append(nlp.X[i])
                     u.append(nlp.U[i])
 
+            elif node == Node.PENULTIMATE:
+                t.append(nlp.ns - 1)
+                x.append(nlp.X[-2])
+                u.append(nlp.U[-1])
+
```

Comment:
Please add a test for this part

Reply:
Should I add an example ?

Reply:
I don't feel it is necessary, but if you feel like to, I won't prevent you from doing that haha!